tests/integration-tests/conftest.py

```diff
@@ -210,6 +210,10 @@ def pytest_addoption(parser):
         help="The build number passed from the testing pipelines",
         default=0,
     )
+    parser.addoption(
+        "--proxy-stack",
+        help="Name of CFN stack providing a Proxy stack to be used for testing Proxy feature.",
```

Comment:
Nit: providing a proxy stack => providing a proxy environment
"to be used for testing Proxy feature." => can be omitted. The proxy stack could be used also for other tests in future.

Reply:
Done!